enterprise/server/remote_execution/operation/operation.go

```diff
@@ -96,10 +94,12 @@ func PublishOperationDone(stream StreamLike, taskID string, adInstanceDigest *di
 	return finalErr
 }
 
-func ExecuteResponseWithResult(ar *repb.ActionResult, summary *espb.ExecutionSummary, code codes.Code) *repb.ExecuteResponse {
-	rsp := &repb.ExecuteResponse{
-		Status: &statuspb.Status{Code: int32(code)},
-	}
+func ExecuteResponseWithCachedResult(ar *repb.ActionResult) *repb.ExecuteResponse {
```

Comment:
nit: could you comment what this does and the one below it (ExecuteResponseWithResult) too?

Reply:
Done